test/integration/fixtures/jwt.txt

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJSUzI1NiIsImtpZCI6InRlc3Qta2V5LTEifQ.eyJzY29wZSI6Im9wZW5pZCBwcm9maWxlIGVtYWlsIiwiY2xpZW50X2lkIjoidGVzdC1jbGllbnQtaWQiLCJpYXQiOjE3NTE0MzY4ODYsImlzcyI6Imh0dHBzOi8vYXV0aC5jaXZpYy5jb20vb2F1dGgvIiwic3ViIjoidGVzdC11c2VyLTEyMyIsImV4cCI6MjA2NzAxMjg4Nn0.H55b0qGZegRJdArFj6Z6EPcb8epI2tF2nE9k4RzjbkiKwtbBNXZLFXiI_BYxp6zgUqAxFaAMtpCwgA7wuoDvMw1MVvcXqXyAbVUOpz-ls0qlq8dePk-NKQevJCLIWaqL9eogw_cLMknR4i-fkwsWaGdziybMyhJVLE-qxuYHX4QyojQ6BBu3H-4H1QCi1sUXlfcFgFAZEYQfe3DM40thcQjZb6D33_Kpw7TgAAMmL-tS3ti0m4QK2rGdbAtpJDaPayqMNM4yp5W2t_bNdMHv52Puf7ZMHqwWeqYfcpeMri_gUz_OUOe6x02YZdOfkqP0uL9eXC6yk9U3u0QZIya0Zg
```

Comment:
Is that a real signed access_token from auth.civic.com until 2035? Any possibility to create that dynamically for the tests?